sema4ai/src/sema4ai_code/workspace_manager.py

```diff
@@ -120,25 +123,66 @@ def _get_agent_package_organizations(
             if not org_directory.is_dir():
                 continue
 
+            action_packages: DefaultDict[
+                str, List[Tuple[str, LocalAgentPackageOrganizationInfoDict]]
+            ] = defaultdict(list)
+
             # Organization name is the name of the directory the Action Package exists in.
             organization_name = os.path.basename(org_directory)
-            organization: LocalAgentPackageOrganizationInfoDict = {
+
+            organization = {
                 "name": organization_name,
                 "actionPackages": [],
             }
 
-            for sub in org_directory.iterdir():
-                action_package_metadata = self._get_package_metadata(
-                    [PackageYamlName.ACTION], sub, curr_cache, new_cache
+            for action_package_metadata in self._get_action_package_from_dir(
```

Comment:
I think that the `log.info` occurrences added should be changed to `log.debug` as they seem pretty verbose (I'd even be fine removing it...).